src/sentry/search/eap/columns.py

```diff
@@ -55,13 +58,31 @@ def proto_definition(self) -> AttributeAggregation | AttributeKey:
         else:
             return AttributeKey(
                 name=self.internal_name,
-                type=(
-                    self.internal_type
-                    if self.internal_type is not None
-                    else constants.TYPE_MAP[self.search_type]
-                ),
+                type=self.proto_type,
             )
 
+    @property
+    def proto_type(self) -> AttributeKey.Type.ValueType:
```

Comment:
i sorta want to call this `rpc_type` so it doesn't read as `prototype`... but to be consistent with `proto_definition` calling it `proto_type`